src/components/ListItem.js

```diff
@@ -39,22 +41,33 @@ const ListItem = ({component}) => {
             {component.shortDescription}
           </Col>
         </Row>
-      </div>
         <Row className="mb-2">
           <Col sm={3} lg={2}>
             <strong>Repo Link:</strong>
           </Col>
           <Col sm={9} lg={10}>
-            <a href={component.gitHubLink} title="Repository Link"
-              target="_blank" >{component.gitHubLink}
+            <a
+              href={component.gitHubLink}
+              title="Repository Link"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {component.gitHubLink}
             </a>
           </Col>
         </Row>
-      <hr />
-      {tags}
-    </Card.Body>
-  </Card>
-)
+        <hr />
+        {renderTags}
+        <Button
+          className="float-right"
+          variant="primary"
+          onClick={handleClick}
+        >
```

Comment:
I feel the card should be clickable over a button, but we can have Roop review our design ideas 😄